app/concerns/agent_controller_concern.rb

```diff
@@ -39,45 +39,47 @@ def validate_control_action
 
   def control!
     control_targets.each { |target|
-      begin
-        case control_action
-        when 'run'
-          case
-          when target.cannot_be_scheduled?
-            error "'#{target.name}' cannot run without an incoming event"
-          when target.disabled?
-            log "Agent run ignored for disabled Agent '#{target.name}'"
-          else
-            Agent.async_check(target.id)
-            log "Agent run queued for '#{target.name}'"
-          end
-        when 'enable'
-          case
-          when target.disabled?
-            target.update!(disabled: false)
-            log "Agent '#{target.name}' is enabled"
-          else
-            log "Agent '#{target.name}' is already enabled"
-          end
-        when 'disable'
-          case
-          when target.disabled?
-            log "Agent '#{target.name}' is alread disabled"
+      interpolate_with('target' => target) {
```

Comment:
I don't have a particular preference here against "the" style guide, except that I'd prefer using brackets when a method call with a block is an expression rather than a construct.